packages/backend/src/persistence/getDatabase.ts

```diff
@@ -97,6 +97,7 @@ export interface IDatabase extends IAccountDatabase {
     addConversation: (
         ensName: string,
         encryptedContactName: string,
+        encryptedProfileLocationL: string,
```

Comment:
typo , please change **encryptedProfileLocationL**  to  **encryptedProfileLocation**